flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -1090,9 +1093,9 @@ static void altitude_hold_desired(ManualControlCommandData * cmd, bool flightMod
 		if (cmd->Throttle > DEADBAND_HIGH) {
 			climb_rate = expo3((cmd->Throttle - DEADBAND_HIGH) / (1.0f - DEADBAND_HIGH), altitude_hold_expo) *
 		                         altitude_hold_maxclimbrate;
-		} else if (cmd->Throttle < DEADBAND_LOW && altitude_hold_maxclimbrate > MIN_CLIMB_RATE) {
+		} else if (cmd->Throttle < DEADBAND_LOW && altitude_hold_maxdescentrate > MIN_CLIMB_RATE) {
```

Comment:
I may be being thick headed.  What's the reason for the second half of this conditional?